octis/preprocessing/preprocessing.py

```diff
@@ -183,10 +187,15 @@ def preprocess_dataset(self, documents_path, labels_path=None, multilabel=False)
                 labels = [
                     line.strip()
                     for line in open(labels_path, 'r').readlines()]
-
+            
             vocab = set(vocabulary)
+            
+
```

Comment:
Generally don't want to have more than one blank line inside a function - two blank lines is reserved for spacing out classes, imports, etc.